content/docs/pipelines/sdk/client_usage.md

```diff
@@ -0,0 +1,40 @@
++++
+title = "Use Kubeflow Pipelines API Client"
+description = "KFP SDK version >= 0.2.5 is required"
```

Comment:
Ohh, you wrote it here. I think description usually refer to a summary of this page (that has more words than a title), rather than detailed requirements.

Reply:
I kind of abused the field. What I am thinking is, like some other doc page, has some warning banner at the top of the page (under title) indicating that the warning applies to the whole page.

Reply:
Since >=0.2.5 is a prerequisites and a must-have, I would like to put it as early as possible

Reply:
I used the suggested line for description. I have another question. Do we have banner like thing to somehow warn users about the version requirement? The reason is that we put this help doc on Kubeflow site while Kubeflow is (consistently) using KFP's old version due to the release process. So this warning message is kind of important.